packages/modeling/src/maths/mat4/isOnlyTransformScale.test.js

```diff
@@ -1,9 +1,11 @@
 const test = require('ava')
 
+const { TAU } = require('../constants')
+
 const { isOnlyTransformScale, create, fromTranslation, fromTaitBryanRotation, fromScaling, invert, multiply } = require('./index')
 
 test('mat4: isOnlyTransformScale() should return true for right angles', (t) => {
-  let someRotation = fromTaitBryanRotation(create(), Math.PI, 0, 0)
+  let someRotation = fromTaitBryanRotation(create(), TAU / 2, 0, 0)
```

Comment:
This is bad for performance. Division is a very costly operation. Replacing 2π with ⲧ makes sense though.

Reply:
In case the TAU/2 expression is kept, it is much faster to multiply by 0.5  than divide by 2.